canvas_modules/common-canvas/__tests__/test_resources/json/form-structure2-test.json

```diff
@@ -261,6 +261,10 @@
                           "valueLabels": [
                             "Ascending",
                             "Descending"
+                          ],
+                            "valueDescs": [
```

Comment:
This looks like an extra tab was added here.

Reply:
Removed it.